{
	"schema_version": "1.6.0",
	"id": "MAL-2024-5643",
	"modified": "2024-10-24T01:01:59Z",
	"published": "2024-06-25T13:39:28Z",
	"summary": "Malicious code in pyantimalware (PyPI)",
	"details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
	"affected": [
		{
			"package": {
				"ecosystem": "PyPI",
				"name": "pyantimalware",
				"purl": "pkg:pypi/pyantimalware"
			},
			"versions": [
				"1.5"
			],
			"database_specific": {
				"source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/pyantimalware/MAL-2024-5643.json"
			}
		}
	],
	"credits": [
		{
			"name": "ReversingLabs",
			"contact": [
				"https://www.reversinglabs.com"
			],
			"type": "FINDER"
		}
	],
	"database_specific": {
		"malicious-packages-origins": [
			{
				"id": "RLMA-2024-04425",
				"import_time": "2024-06-28T02:49:58.723176505Z",
				"modified_time": "2024-06-25T13:39:28Z",
				"sha256": "9eee953d360166fde58ebd867387ea522b091885f6201db7aca8b60c8c90cfbc",
				"source": "reversing-labs",
				"versions": [
					"1.5"
				]
			},
			{
				"id": "RLUA-2024-08860",
				"import_time": "2024-10-24T00:59:46.539450525Z",
				"modified_time": "2024-10-16T14:47:12Z",
				"sha256": "bf5d09f1c0c1fd00a66bd47c2075409691130d1f99d695614681b5d65d30e64c",
				"source": "reversing-labs"
			}
		]
	}
}
